test/EVM/Test/Utils.hs

```diff
@@ -59,7 +59,6 @@ testOpts solvers root buildOutput match maxIter allowFFI rpcinfo = do
     , askSmtIters = 1
     , smtTimeout = Nothing
     , solver = Nothing
-    , verbose = Just 1
```

Comment:
This is now `conf` and in cli.hs is set to by default 1. In API library use mode it's set to 0 by default.